.github/actions/snapshoty/index.js

```diff
@@ -24,18 +24,28 @@ async function run() {
       '-e', `GCS_PRIVATE_KEY_ID=${gcsPrivateKeyId}`,
       '-e', `GCS_PROJECT=${gcsProject}`
     ]
+
+    // GCS env vars are secrets
+    for (secret of [gcsClientEmail, gcsPrivateKey, gcsPrivateKeyId, gcsProject]) {
+        core.setSecret(secret);
+    }
+
+    // Forward env vars
     Object.keys(process.env).forEach(function (key) {
       if (key.startsWith("GITHUB_") || key.startsWith("RUNNER_")) {
         let value = process.env[key];
         args.push('-e', `${key}=${value}`);
-        core.setSecret(value);
+
+        if (key === 'GITHUB_TOKEN') {
```

Comment:
Note: Only the github token should be considered as a secret.
It should already be a secret.